google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -37,6 +38,11 @@
   // set the option ThrowNotFound when you want to throw the exception when the value not found
   private boolean setThrowNotFound;
 
+  @Override
+  public void close() throws Exception {
+    System.out.println("Closed BigQuery Resource");
```

Comment:
I imagine there are some resources to actually clean up. For example, in Python there are two long-running sockets (one for handling auth token updates, one for the actual API requests) that need to be explicitly closed.

Reply:
+1 - Also, we probably don't want to print to `stdout`. If we do want to print this anywhere, we want to use a debug logger.